wgpu-native/src/device.rs

```diff
@@ -1192,9 +1194,7 @@ pub fn device_create_bind_group<B: GfxBackend>(
                         binding_model::BindingType::ReadonlyStorageBuffer => {
                             (BIND_BUFFER_ALIGNMENT, resource::BufferUsage::STORAGE_READ)
                         }
-                        binding_model::BindingType::Sampler
-                        | binding_model::BindingType::SampledTexture
-                        | binding_model::BindingType::StorageTexture => {
+                        _ => {
```

Comment:
please don't change this - it's unrelated, and we explicitly want to enumerate the variants